documentation/staging/content/managing-domains/accessing-the-domain/access-admin-console-em.md

```diff
@@ -0,0 +1,210 @@
+---
+title: "Access the Administration Console"
+date: 2019-02-23T17:39:15-05:00
+draft: false
+weight: 1
+description: "Access the WebLogic Admin Console to manage domains running in Kubernetes."
```

Comment:
"Access the" --> "Use the ..."

Reply:
"Admin" --> "Administration" throughout (@rosemarymarano will know for sure, or compare to the WL doc)

Reply:
@ankedia @tbarnes-us Yes, absolutely.

Reply:
Changed.